Web UI/CognitiveSearch.UI/CognitiveSearch.UI.csproj

```diff
@@ -2,9 +2,11 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <ApplicationInsightsResourceId>/subscriptions/789908e0-5fc2-4c4d-b5f5-9764b0d602b3/resourcegroups/knowledgemining-og-rg/providers/microsoft.insights/components/kminingdemoapp</ApplicationInsightsResourceId>
```

Comment:
do we need to put the sub-id and rg name here, it probably is needed by our deployed app but not here suggest removing it.

Reply:
removed this, and replaced it with "_subsriptionId_" and "_resourceGroupName_"